### PR TITLE
Update compare-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line about using m365 groups for permissions to a site

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8111

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/create-groups/compare-groups.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Group members can send as or send on behalf of the group email address if this h
 
 Microsoft 365 Groups don't support nesting with other Microsoft 365 Groups or with distribution or security groups.
 
+Microsoft 365 Groups can be added to to one of the three SharePoint groups (owners, members or visitors) to give people permissions to the site.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/compare-groups.md]:
```suggestion
Microsoft 365 Groups can be added to one of the three SharePoint groups (Owners, Members, or Visitors) to give people permissions to the site.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**